File: Week1/1-4_최윤성_20210705.cpp
#include <iostream>
#include <vector>

using namespace std;

int main() {
    int testcase;
    cin >> testcase;
    vector<float> ratio;
    for(int i = 0; i < testcase; i++) {
        int count, stud = 0;
        cin >> count;
        float sum = 0;
        vector<int> scores;
        for(int j = 0; j < count; j++) {
            int score;
            cin >> score;
            scores.push_back(score);
            sum += score;
        }
        sum /= count;
        for(int score : scores)
            if(score > sum) stud++;
        ratio.push_back((float)stud / count);
    }

    for(float stud : ratio) {
        cout << fixed;
        cout.precision(3);
        cout << stud * 100 << "%" << endl;
    }
}

File: Week1/1-4_최윤성_20210705.py
testcase = int(input())
ratio = []
for i in range(testcase):
    scores = list(map(int, input().split()))
    ratio.append(len(list(filter(lambda x : x > sum(scores[1:])/scores[0], scores[1:]))) / scores[0])
for i in ratio: print("%.3f%%" % (i * 100))